### PR TITLE
Update ch.bafu.waldreservate

### PR DESCRIPTION
**Pull request by faselm**
Test link: https://mf-chsdi3.dev.bgdi.ch/ltfas_bafu_waldreservate/shorten/8ff5f553da

### CODE REVIEW

File: chsdi/models/vector/bafu.py [hunk 1]
```diff
@@ -2160,6 +2160,11 @@ class Waldreservate(Base, Vector):
     gisflaeche = Column('obj_gisflaeche', Float)
     gesflaeche = Column('obj_gesflaeche', Float)
     mcpfe = Column('mcpfe_class', Unicode)
+    mcpfe_class_de = Column('mcpfe_class_de', Unicode)
+    mcpfe_class_fr = Column('mcpfe_class_fr', Unicode)
+    mcpfe_class_it = Column('mcpfe_class_it', Unicode)
+    mcpfe_class_en = Column('mcpfe_class_en', Unicode)
+    mcpfe_class_rm = Column('mcpfe_class_rm', Unicode)
```
Comment (b4l) [inline chsdi/models/vector/bafu.py:2167]: Why specify rm and in the mako you map rm to de?

Comment (faselm) [reply chsdi/models/vector/bafu.py:2167]: Yep, it can be removed in the model...

**Review by ltclm — CHANGES_REQUESTED** [review]
thanks for this @faselm !
Please remove mcpfe_class_rm from model and table if its not used / mapped to de in the htmlpopup service.

File: chsdi/templates/htmlpopup/bafu_waldreservate.mako [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-    <tr><td class="cell-left">${_('ch.bafu.waldreservate.name')}</td>                         <td>${c['attributes']['name'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bafu.waldreservate.gisflaeche')}</td>                   <td>${round(c['attributes']['gisflaeche'],2) or '-'}</td></tr>
+
+    <%
+       lang = 'de' if lang in ('de', 'rm') else lang
+       class_text = 'mcpfe_class_%s' % lang
+    %>
+
+    <tr><td class="cell-left">${_('ch.bafu.waldreservate.objnummer')}</td><td>${c['attributes']['objnummer'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bafu.waldreservate.name')}</td><td>${c['attributes']['name'] or '-'}</td></tr>
```
Comment (rebert) [inline chsdi/templates/htmlpopup/bafu_waldreservate.mako:11]: I'm not a specialist in Rumantsch Grischun - but Name is the only declaration being translated - and that is not consistent. This time Name is translated to Num

**Review by rebert — CHANGES_REQUESTED** [review]
Good. Only a detail in a translation

**Comment by faselm** [comment]
rm translation also fixed here...

**Review by ltclm — CHANGES_REQUESTED** [review]
looks good to me.
the attribute mcpfe_class is obviously only used in the queryable attributes list anymore. please ask @mariokeusen if we can remove it from chsdi entirely.

**Review by ltclm — COMMENTED** [review]
you would have to remove it from  `__queryable_attributes__` too, but please ask mario first.

**Comment by ltclm** [comment]
please ask @mariokeusen if we can completely remove this functionality from this layer:
image

and then remove queryable_attributes, this feature is not used at all for this layer.

**Review by ltclm — APPROVED** [review]

**Pull request merged**